rosidl_generator_py/rosidl_generator_py/__init__.py

```diff
@@ -21,6 +21,7 @@
 
 try:
     from .generate_py_impl import generate_py
+    assert generate_py
```

Comment:
@wjwwood I believe this only works with [flake8](https://pypi.python.org/pypi/flake8)